pkg/ccl/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -396,10 +396,10 @@ func TestPreviouslyInterestingTables(t *testing.T) {
 				sqlA, tableName, numInserts, nil)
 			require.NoError(t, err)
 			streamStartStmt := fmt.Sprintf(
-				"SELECT crdb_internal.start_logical_replication_job('%s', ARRAY['%s'])",
-				dbAURL.String(), tableName)
+				"CREATE LOGICAL REPLICATION STREAM FROM $1 ON TABLE %s INTO TABLE %s",
+				tableName, tableName)
```

Comment:
I agree with the sentiment you've expressed elswhere that `ON` and `FROM` seem backwards to my reading. `FROM <table> ON <cluster>`.

Reply:
Let's get @alicia-l2's confirmation and then I'll open a follow-up PR swapping the order if she agrees too.

Reply:
I'm fine with that

Reply:
great, I'll do it in a follow-up.

Reply:
follow-up is here: https://github.com/cockroachdb/cockroach/pull/126402